core/domain/exp_jobs_one_off.py

```diff
@@ -369,6 +369,15 @@ def map(item):
             yield ('ERROR get_exp_from_model: exp_id %s' % item.id, str(e))
             return
 
+        # Commit commands which are required to generate state id mapping for
```

Comment:
Just to check, do you need 'create exploration' too or similar (to initialize the list of states with the initial default list of states for version 1)?

Guessing not but just wanted to double check.